src/main/java/edu/hawaii/its/groupings/configuration/AppConfig.java

```diff
@@ -2,9 +2,18 @@
 
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
 
 @Configuration
 @ComponentScan(basePackages = "edu.hawaii.its")
+@PropertySources({
+        @PropertySource("classpath:custom.properties"),
+        @PropertySource(value = "file:${user.home}/.${user.name}-conf/uh-groupings-ui-overrides.properties",
+                ignoreResourceNotFound = true),
+        @PropertySource(value = "file:${user.home}/.${user.name}-conf/myiam-overrides.properties",
+                ignoreResourceNotFound = true)
+})
```

Comment:
Instead of defining it in AppConfig, modify the application.properties files. There are 7 different application properties files (application.properties, application-integrationTest.properties, etc...)

Reply:
@JorWo Hi Jordan, 
I completed 
- deleted the myiam.overrides.properties file.
- edited 7 application properties files.
Thank you.